INSTALL.md

```diff
@@ -105,7 +111,7 @@ make opamdep
 
 ### If you have opam package manager already installed
 You can try installing the Scilla dependencies using the instructions above, but skipping the initialization step.
-If `opam` reports a dependency conflict, one way out might be creating yet another opam switch and 
```

Comment:
Ah, I didn't mean to change this line, my editor fixes trailing spaces automatically.

Reply:
No worries :)